dataset_generation/hi_wikisplit/Scraper.py

```diff
@@ -0,0 +1,174 @@
+import logging
```

Comment:
also, can we rename the file to wiki_scraper.py? (all small letters to sort of follow the same convention throughout)

Reply:
yes, i'll rename the file.